### DISCUSSION TITLE
How to customise resource requests and limits for __ASSET_JOB?

### DISCUSSION THREAD

**Discussion by bolinzzz** [dagster-io/dagster#18627] [Q&A] [answered]
## My current setup
I am relying on auto-materialisation to materialise most of my assets. Certain assets are more memory hungry and require higher memory requests and limits.

I am tagging those memory hungry assets as such
```
{
  "container_config": {
    "resources": {
      "limits": {
        "memory": "15Gi"
      },
      "requests": {
        "memory": "10Gi"
      }
    }
  },
  "pod_spec_config": {
    "nodeSelector": {
      "type": "etl-spot"
    }
  }
}
```
In order for these configurations to be applied, I created jobs that are using `k8s_job_executor`.

## What I want to achieve
At the moment, is there a way for `__ASSET_JOB` which are adhoc jobs to auto-materialise assets to pick up and apply resource requests and limits that I define at the asset level?

**Comment by sryza** [comment, marked as answer]
@bolinzzz this is not currently possible, but being tracked here: https://github.com/dagster-io/dagster/issues/8979.
